pkg/collector/adapters/config_to_probe_test.go

```diff
@@ -0,0 +1,173 @@
+package adapters_test
```

Comment:
You can use the same package name as the code under test. This way, you can use the non-exported errors. Use a different package if you want to exercise only the public API.

Reply:
Sounds good, have updated accordingly.